### ISSUE TITLE
Support for versioned Manifest.toml files (>= julia 1.11)

### ISSUE THREAD

**Issue by Sbozzolo** [CliMA/PkgDevTools.jl#11]
Unfortunately, Julia 1.10 and Julia 1.11 have incompatible Manifests, which leads to the need for versioning the Manifest with `Manifest-v1.11.toml`. It would be nice if `PkgDevTools` supported versioned Manifests and updated the correct files depending on the version of Julia used.

**Comment by charleskawczynski** [comment]
Ah, yeah, good point and good idea

**Comment by charleskawczynski** [comment]
Actually, this does work, but it preserves the version. So, launching with `julia +1.10 --project` will update the 1.10 (and down) manifests, and `julia +1.11 --project` will update the 1.11 manifests.

**Comment by charleskawczynski** [comment]
I think what might be nice would be to ask users which julia versions to upgrade, so that it can loop over the versions.

**Comment by charleskawczynski** [comment]
I'm going to update the title to reflect this.

**Comment by charleskawczynski** [comment]
In ClimaAtmos, we have multiple versions, so it'd be nice to not have to run / select this twice.

**Comment by Sbozzolo** [comment]
> Actually, this does work, but it preserves the version. So, launching with `julia +1.10 --project` will update the 1.10 (and down) manifests, and `julia +1.11 --project` will update the 1.11 manifests.

Yes, this is what I've been doing, so I don't know exactly what I meant when I opened the issue.

Maybe the problem I found was  when you only have a Manifest.toml (and not a Manifest-v1.11.toml). In that case, I think the desired outcome is for a Manifest-v1.11 to be created if you are running Julia 1.11 (and up), and not updating the manifest to 1.11 (because that would break compatibiity with previous versions of julia)

> In ClimaAtmos, we have multiple versions, so it'd be nice to not have to run / select this twice.

Yes, I agree.
